101-cses-multi-role-rdp/azuredeploy.json

```diff
@@ -0,0 +1,180 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "cloudServiceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the cloud service (extended support)"
+      }
+    },
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "Location of the cloud service (extended support)"
+      }
+    },
+    "deploymentLabel": {
+      "type": "string",
+      "metadata": {
+        "description": "Label of the deployment"
+      }
+    },
+    "packageSasUri": {
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts